pkg/prove/querier.go

```diff
@@ -54,3 +58,93 @@ func QueryTxInclusionProof(_ sdk.Context, path []string, req abci.RequestQuery)
 
 	return rawTxProof, nil
 }
+
+const ShareInclusionQueryPath = "shareInclusionProof"
+
+// QueryShareInclusionProof defines the logic performed when querying for the inclusion
+// proofs of a set of shares to the data root.
+// the shares range should be appended to the path.
```

Comment:
[nit] sentence case godoc comments

```suggestion
// QueryShareInclusionProof defines the logic performed when querying for the
// inclusion proofs of a set of shares to the data root. The share range should
// be appended to the path. Example path for proving the set of shares [3, 5]:
// custom/shareInclusionProof/3/5
```

Reply:
4595dd2